lib/_stream_readable.js

```diff
@@ -132,9 +135,25 @@ function Readable(options) {
   if (options && typeof options.read === 'function')
     this._read = options.read;
 
+  if (options && typeof options.destroy === 'function')
```

Comment:
Can `options` be `typeof options === 'object'`. I believe by this point we know that it isn't `null`.